solutions/instances/variables.tf

```diff
@@ -226,6 +226,27 @@ variable "profile_attachments" {
   default     = ["IBM Cloud Framework for Financial Services"]
 }
 
+variable "resource_groups_scope" {
+  type        = list(string)
+  description = "The resource group to scope the Security and Compliance Center profile attachments to. If left empty, the default scoped resource group will be set to the existing or new resource group passed to this module. Currently has a limitation of accepting a max number of one resource groups."
```

Comment:
This description is wrong. If not group is specified here it will default to the full account right?

Reply:
yes, updated